star-ros2/src/star_spi_bridge/include/star_spi_bridge/star_spi_driver_node.hpp

```
@@ -1,7 +1,7 @@
 // Copyright 2026 Locked Inc.
 
-#ifndef STAR_SPI_BRIDGE_INCLUDE_STAR_SPI_BRIDGE_STAR_SPI_DRIVER_NODE_HPP_
-#define STAR_SPI_BRIDGE_INCLUDE_STAR_SPI_BRIDGE_STAR_SPI_DRIVER_NODE_HPP_
+#ifndef STAR_SPI_BRIDGE__STAR_SPI_DRIVER_NODE_HPP_
+#define STAR_SPI_BRIDGE__STAR_SPI_DRIVER_NODE_HPP_
```

Clang (14.0.6)

[warning] declaration uses identifier 'STAR_SPI_BRIDGE__STAR_SPI_DRIVER_NODE_HPP_', which is a reserved identifier

(bugprone-reserved-identifier)